src/tests/multihost/ad/test_adparameters.py

```diff
@@ -135,18 +135,15 @@ def test_0003_bz1421622(self, multihost, adjoin, create_aduser_group):
                 assert False
 
     @pytest.mark.tier1
-    def test_00015_authselect_cannot_validate_its_own_files(self, multihost):
+    def test_00015_authselect_cannot_validate_its_own_files(self, multihost, adjoin):
         """
         :title: authselect: authselect cannot validate its own files
         :id: 67bec814-d67b-4469-9662-58354889d549
         :requirement: IDM-SSSD-REQ :: Authselect replaced authconfig
         :casecomponent: authselect
         :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1734302
         """
-        password = multihost.ad[0].ssh_password
-        client = sssdTools(multihost.client[0])
-        domainname = multihost.ad[0].domainname.strip().upper()
-        client.realm_join(domainname, password)
+        adjoin(membersw='adcli')
```

Comment:
Is adjoin operation getting reverted in testcase-teardown

Reply:
rest looks good

Reply:
The fixture adjoin does that.